packages/compiler/src/render3/view/styling_builder.ts

```diff
@@ -610,3 +614,7 @@ function getStylePropInterpolationExpression(interpolation: Interpolation) {
 function normalizePropName(prop: string): string {
   return hyphenate(prop);
 }
+
+function isCssCustomProperty(name: string): boolean {
```

Comment:
I think it'd be helpful to have a link here as well:
```suggestion
// Checks whether property name is a custom CSS property.
// See: https://www.w3.org/TR/css-variables-1/#defining-variables
function isCssCustomProperty(name: string): boolean {
```